source/project-stories/petition-map.erb

```diff
@@ -0,0 +1,152 @@
+
+quote_3:
+  quote: "Since the EU Referendum petition, by popular demand the map has been updated with a new colouring option – weighting each constituency with its population. Hovering over each constituency also shows you the percentage of constituents that have signed. These new features make Petition Map an even more useful tool for examining regional engagement with politics."
+  source: "Cale Tilford, Developer, Unboxed"
+
+team:
+  title: "Unboxed team"
+  people:
+  - full_name: "Cale Tilford"
+    image: "/project-stories/petition-map/team/cale.png"
+    link: "/people#cale-tilford"
+
+site:
+  link: "http://petitionmap.unboxedconsulting.com/"
+  text: "Visit the Petitions Map"
+
+links:
+  - title: "Public sector collaboration with GDS to relaunch Petitions platform for UK citizens"
+    source: ""
+    type: "project story"
+    link: "/project-stories/petitions/"
+
+  - title: "Record-breaking EU 2nd referendum petition hits 3.9 million signatures and rising"
+    source: ""
+    type: "news"
+    link: "/news/record-breaking-eu-2nd-referendum-petition-hits-3-9-million-signatures-and-rising-most-popular-parliament-petition-ever/"
+
+---
+
+<%= partial 'project-stories/page_header' %>
+
+<div class="container" data-header-waypoint>
+  <%= partial 'project-stories/full_image', locals: { section: current_page.data.header } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.introduction } %>
+
+  <section class="highlight--petition-map">
+    <h2 class="highlight__title">
+      <%= current_page.data.statistics_1.title %>
+    </h2>
+
+    <div class="highlight__tile--two">
+      <p class="highlight__large-text"><%= current_page.data.statistics_1.number %></p>
+      <p class="highlight__small-text"><%= current_page.data.statistics_1.text %></p>
+      <p class="highlight__large-text"><%= current_page.data.statistics_2.number %></p>
+      <p class="highlight__wide-text"><%= current_page.data.statistics_2.text %></p>
+    </div>
+  </section>
+  
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.about } %>
+
+	<blockquote class="quote">
+	  <aside class="quote__right-section">
+	    <p class="quote__text--centered">
+	      <%= current_page.data.quote_1.quote %>
+	    </p>
+	    <p class="quote__source">
+	      <%= current_page.data.quote_1.source %>
+	    </p>
+	  </aside>
+	  <img class="quote__source-image" src="<%= current_page.data.quote_1.image %>" alt="<%= current_page.data.quote_1.source %>"/>
+	</blockquote>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.open_source } %>
+
+  <%= partial 'project-stories/quote', locals: { section: current_page.data.quote_2 } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.public_following } %>
+
+  <%= partial 'project-stories/quote', locals: { section: current_page.data.quote_3 } %>
+
+  <%= partial 'project-stories/project_team_single' %>
```

Comment:
The idea behind these partials is that we shouldn't care about the size of the data provided to them, they should just render. The responsibility for how the team renders is down to the `project_team` partial. By introducing a new partial, specifically for one team member, breaks this pattern as we could just change the original partial to produce different HTML based on the team size.

We should remove this `project_team_single` file and alter `project_team` and/or the respective SCSS.

There are two options that I think we have:

1. Leave the HTML as it is and overwrite the styles specifically for `.project-team__container--1` (mostly that from what is produced from `.project-team__headshot-container`)

2. Alter the HTML to produce the desired classes for a single centered image.

The latter would be clearer and we wouldn't need to unnecessary override the CSS styles seeing as we control the HTML generation.

I'd recommend something along the following lines:

```CSS
/* source/assets/stylesheets/styles/_project_team.scss */

.project-team__container--1 {
  display: block;
  margin: 0 auto;
  max-width: 160px;
}
```

```Erb
<!-- source/project-stories/_project_team.erb -->

<% team_size = current_page.data.team.people.size %>
<% link_class = "project-team__container--#{team_size}" %>
<% link_class << " project-team__headshot-container" if team_size > 1 %>

<% current_page.data.team.people.each do |person| %>
  <% if person.link %>
    <a class="<%= link_class %>" href="<%= person.link %>">
      <img class="project-team__headshot" src="<%= person.image %>" alt="<%= person.full_name %>" title="<%= person.full_name %>">
    </a>
  <% else %>
    <div class="<%= link_class %>">
      <img class="project-team__headshot" src="<%= person.image %>" alt="<%= person.full_name %>" title="<%= person.full_name %>">
    </div>
  <% end %>
<% end %>
```